compose/transfer/compose.yml

```diff
@@ -134,6 +134,48 @@ services:
             - capabilities: ["gpu"]
               driver: nvidia
               device_ids: ["${GPU:-0}"]
+
+  katago-raw:
```

Comment:
This largely duplicates `katago`. Can we define an alias for `volumes:` using `&` like described in https://stackoverflow.com/questions/6651275/what-do-the-mean-in-this-database-yml-file so we can just copy that over with `<<: *` syntax as I think that part is verbatim? Or maybe we could alias the whole `katago:` config, and then just replace the parts of it we need which I think is just the `profiles:` and `command:` part, as I think YAML should merge the dicts automatically.